src/main/java/Task.java

```diff
@@ -0,0 +1,36 @@
+public class Task {
+    private String description;
+    private boolean isCompleted;
+
+    // Constructor
+    public Task(String description) {
+        this.description = description;
+        this.isCompleted = false; // Tasks start as not completed by default
+    }
+
+
+    // Mark the task as completed
+    public void completeTask() {
+        this.isCompleted = true;
+    }
+
+    // Mark the task as not completed (or "uncheck" the task)
+    public void uncheckTask() {
+        this.isCompleted = false;
+    }
```

Comment:
`completeTask` and `uncheckTask` are both setters for the boolean `isCompleted`, perhaps the method names can be changed to make them sound similar and also adhere to coding standards for boolean setters?

Reply:
Yes, I agree as well. Some level of similarity in the setters would also make it easy for code readability.